app/Child.php

```diff
@@ -2,10 +2,26 @@
 
 namespace App;
 
+use Carbon\Carbon;
 use Illuminate\Database\Eloquent\Model;
-
+use Log;
 class Child extends Model
```

Comment:
Boring - but do we want an extra line here?

Reply:
Actually - are you using Log anymore or was for debugging?

Reply:
Will amend.